benchexec/tablegenerator/test_integration/__init__.py

```diff
@@ -12,16 +12,21 @@
 import sys
 import tempfile
 import unittest
+import platform
+import re
 
 import benchexec
 import benchexec.util
+import benchexec.tablegenerator.util
 
 sys.dont_write_bytecode = True  # prevent creation of .pyc files
 
 here = os.path.relpath(os.path.dirname(__file__))
 base_dir = os.path.join(here, "..", "..", "..")
 bin_dir = os.path.join(base_dir, "bin")
-tablegenerator = os.path.join(bin_dir, "table-generator")
+tablegenerator = [os.path.join(bin_dir, "table-generator")]
+if platform.system() == "Windows":
+    tablegenerator = [sys.executable] + tablegenerator
```

Comment:
I would suggest to drop the platform check. Using `sys.executable` actually seems like a good idea for Linux as well because of virtual envs.